ansible_role/tasks/main.yml

```diff
@@ -74,6 +74,8 @@
     - name: broker.route.yaml
       apply: "{{ ('route.openshift.io' in lookup('k8s', cluster_info='api_groups')) | bool }}"
     - name: broker-client.secret.yaml
+    - name: broker-service-ca-bundle.configmap.yaml
+      apply: "{{ ('route.openshift.io' in lookup('k8s', cluster_info='api_groups')) | bool }}"
```

Comment:
Is there a more specific API (like one provided by the cert-signer operator) that we could look for?

Reply:
Good point, I was hoping you would chime in on this. I will check on that.

Reply:
Will there ever not be a cert signer operator running in new versions? I think it's always there in Openshift?

Reply:
My concern is when this runs in Kubernetes or any older version of OpenShift where nothing will happen when you create the configmap and the resulting `(Cluster)ServiceBroker` we create will try and use values from a configmap that will never be filled in.

Reply:
This version of the role is for 4.0 right? We have branches in release-1.3, release-1.2, etc. with roles appropriate to those versions, correct?

Having said that, for kubernetes, perhaps:
```
ok: [localhost] => (item=certificates.k8s.io) => {
    "certificates.k8s.io": "VARIABLE IS NOT DEFINED!",
    "item": "certificates.k8s.io"
```

Reply:
Maybe
```
ok: [localhost] => (item=servicecertsigner.config.openshift.io) => {
    "item": "servicecertsigner.config.openshift.io",
    "servicecertsigner.config.openshift.io": "VARIABLE IS NOT DEFINED!"
}

```

Reply:
@jmontleon @djzager updated to look for `servicecertsigner.config.openshift.io` apigroup